### PR TITLE
Update capacities for France

### PR DESCRIPTION
**Pull request by veqtrus**
pumped hydro

Using 2020 [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show?name=&defaultValue=false&viewType=TABLE&areaType=BZN&atch=false&dateTime.dateTime=01.01.2020+00:00|UTC|YEAR&dateTime.endDateTime=01.01.2021+00:00|UTC|YEAR&area.values=CTY|10YFR-RTE------C!BZN|10YFR-RTE------C&productionType.values=B01&productionType.values=B02&productionType.values=B03&productionType.values=B04&productionType.values=B05&productionType.values=B06&productionType.values=B07&productionType.values=B08&productionType.values=B09&productionType.values=B10&productionType.values=B11&productionType.values=B12&productionType.values=B13&productionType.values=B14&productionType.values=B20&productionType.values=B15&productionType.values=B16&productionType.values=B17&productionType.values=B18&productionType.values=B19) data, except I left geothermal as is. Unknown includes marine+other.

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -1868,17 +1868,17 @@
...
-      "coal": 2968,
-      "gas": 12156,
+      "biomass": 1578,
+      "coal": 2977,
+      "gas": 12238,
       "geothermal": 16,
-      "hydro": 24086,
-      "hydro storage": 1728,
+      "hydro": 16947,
+      "hydro storage": 4656,
```
Comment (jarek) [inline config/zones.json:1876]: These are quite a large change, has something been reclassified?

Also did solar really go down by a gigawatt?

Comment (veqtrus) [reply config/zones.json:1876]: Well, there is evidently more hydro storage, as can be seen in the screenshot.

Comment (jarek) [reply config/zones.json:1876]: And less hydro. So have some plants formerly classified as hydro been reclassified as hydro storage? Electricity Map might need to re-run data archives if so

Comment (Kongkille) [reply config/zones.json:1876]: I'm curious how we ever had hydro storage at 1728, cannot seem to find that number anywhere

**Review by Kongkille — APPROVED** [review]

**Pull request merged**